bridge/svix-bridge/src/config/mod.rs

```diff
@@ -46,8 +46,39 @@ impl Config {
         } else {
             Cow::Borrowed(raw_src)
         };
-        serde_yaml::from_str(&src)
-            .map_err(|e| Error::new(ErrorKind::Other, format!("Failed to parse config: {}", e)))
+        let cfg: Self = serde_yaml::from_str(&src)
+            .map_err(|e| Error::new(ErrorKind::Other, format!("Failed to parse config: {}", e)))?;
+
+        for sc in &cfg.senders {
+            if let Some(tc) = sc.transformation() {
+                crate::runtime::validate_script(tc.source().as_str()).map_err(|e| {
+                    Error::new(
+                        ErrorKind::Other,
+                        format!(
+                            "failed to parse transformation for sender `{}`: {:?}",
```

Comment:
Commonly we validate stuff via proc macros that run during the serde parsing.

In this case, I deferred the work since the underlying types that hold the source code to validate are split out across a few different crates in the workspace and I didn't want to start pushing deno things into more crates.